src/Rollbar.php

```diff
@@ -90,12 +92,49 @@ public static function scope($config)
         return self::$logger->scope($config);
     }
 
-    public static function log($level, $toLog, $extra = array())
+    /**
+     * Logs a message to the Rollbar service with the specified level.
+     *
+     * @param Level|string      $level   The severity level of the message.
+     *                                   Must be one of the levels as defined in
+     *                                   the {@see Level} constants.
+     * @param string|Stringable $message The log message.
+     * @param array             $context Arbitrary data.
+     *
+     * @return void
+     *
+     * @throws InvalidArgumentException If $level is not a valid level.
+     * @throws Throwable Rethrown $message if it is {@see Throwable} and {@see Config::raiseOnError} is true.
+     */
+    public static function log($level, string|Stringable $message, array $context = array()): void
```

Comment:
I guess the reason we don't type the `$level` param here is because of the `psr/log` interface we want to abide, right?

Reply:
That is correct. `psr/log` does not have a type annotation for `$level` so adding one would break the interface implementation.